public/data/akebu6.json

```
@@ -0,0 +1,47 @@
+{
+    "name": "Akebu",
+    "type": "personal",
+    "bio": "Junior in College studying SWE | Aspiring Android & Web  Developer | Open Source | Technical Writing",
+    "avatar": "https://github.com/akebu6.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/akebu6",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/akebu6",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/akebu-simasiku-24186720a/",
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+       {
+        "title": "Started learning Jetpack Compose",
+        "date": "7th May 2022",
+        "icon": "android",
+        "color": "grey",
+        "description": "Started learning Jetpack Compose to compose Android UI"
+      },  
+      {
+        "title": "Completed 100 days of learning Kotlin",
+        "date": "7th May 2022",
+        "color": "grey",
+        "description": "Completed the 100 days of learning Kotlin and Android Development"
+      },
+      {
+        "title": "Started Blogging",
+        "date": "17th May 2022",
+        "icon": "hashnode",
+        "color": "grey",
+        "description": "Started writing articles and tutorials on Kotlin, Design Patterns, C++, Operating Systems and Web Development",
+        "url": "https://akebu6.hashnode.dev/design-patterns-grasp"
+      }
+    ]
+  }
+  
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```